doc/visualizations.rst

```diff
@@ -8,37 +8,83 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and
+input data (`X`, `y`).
+The `from_predictions` method creates a `Display` object from true and predicted
+values (`y_test`, `y_pred`), which is useful when you only want to compute the
+predictions once.
+
+Using `from_predictions` avoids having to recompute predictions, but does not
+automatically resolve some ambiguities. For binary classification, the user must know
+which column corresponds to the positive label (in this case, `y_pred`).
+
+The `Display` object stores the computed values (e.g., metric values or
+feature importance) required for plotting with Matplotlib. These values are the
+derived results after we pass the raw predictions to `from_predictions`, or
+an estimator to `from_estimator`. When the `Display` object is created,
+the plot is also created.
+
+Display objects have a plot method that creates a matplotlib plot once the display
+object has been initialized. Additionally, the plot method allows adding to an existing
```

Comment:
I would add a note like
```suggestion
Display objects have a plot method that creates a matplotlib plot once the display
object has been initialized (note that we recommend that display objects are created
via `from_estimator` or `from_predictions` instead of initialized directly). The plot
method allows adding to an existing
```